kuma/javascript/src/l10n.js

```diff
@@ -14,71 +14,48 @@
  * @flow
  */
 
-// This is the type of the localization data that Django creates for a locale.
-type LocaleData = {
-    catalog: { [string]: string | Array<string> },
-    plural: ?string
-    // The data that Django passes to us also has a formats object
-    // containing template strings for date and time formatting and
-    // defining things like the thousands separator. We don't currently
-    // use them, so they are omitted from this type definition.
-};
+// This is the type of the string catalog that Django creates for a locale.
+type StringCatalog = { [string]: string | Array<string> };
```

Comment:
There are a lot of changes in this file, but most of them are variable and type name changes, and type simplifications like this one.  The change looks bigger that it actually is.